auto_sizing/targets.py

```diff
@@ -108,13 +105,13 @@ def _make_desktop_targets(
                 Segment(
                     name="clients_last_seen_filter",
                     data_source=clients_last_seen,
-                    select_expr=f'COALESCE(MIN(first_seen_date) <= "{add_days(start_date, -28)}", TRUE) AND COALESCE(MIN(days_since_seen) = 0)',
+                    select_expr=f'COALESCE(MIN(first_seen_date) <= "{add_days(start_date, -28)}", TRUE) AND COALESCE(MIN(days_since_seen) = 0)',  # noqa: E501
```

Comment:
`# noqa` should be used sparingly, and I don't think you need it for these line-too-long issues. I think you can use a mix of variables (for repeated project IDs, table names, etc) and strings concatenated across lines (or multiline f-string tuples maybe?).